scripts/analysis.py

```diff
@@ -483,18 +318,16 @@ def get_stockpile(cursor, facility):
 
     Returns
     -------
-    null
-    line plot of stockpile inventory
+    pile_dict: dictionary
+        dictionary with key: facility, value: timeseries of stockpile
     """
-
-    cur = cursor
     pile_dict = collections.OrderedDict()
-    agentid = get_agent_ids(cur, facility)
+    agentid = get_agent_ids(cursor, facility)
     query = exec_string(agentid, 'agentid', 'timecreated, quantity, qualid')
     query = query.replace('transactions', 'agentstateinventories')
-    stockpile = cur.execute(query).fetchall()
-    init_year, init_month, duration, timestep = get_sim_time_duration(cur)
-    stock_timeseries = get_timeseries(stockpile, duration, .001)
+    stockpile = cursor.execute(query).fetchall()
+    init_year, init_month, duration, timestep = get_sim_time_duration(cursor)
+    stock_timeseries = get_timeseries(stockpile, duration, .001, 'TRUE')
```

Comment:
why does this use the string value "TRUE" rather than the python keyword True? Please note that this works because any nonempty string evaluates to True, not because the string is the word true. Use the python boolean objects True and False (not strings).